cxx/pclean/schema_helper_test.cc

```diff
@@ -295,40 +290,35 @@ BOOST_AUTO_TEST_CASE(test_make_hirm_schema_only_final_emissions) {
   T_noisy_relation nr1 = std::get<T_noisy_relation>(tschema["Specialty"]);
   BOOST_TEST(nr1.is_observed);
   BOOST_TEST((nr1.emission_spec.emission == EmissionEnum::bigram_string));
-  // "School", "Physician"  moved to the front of the list.
-  expected_domains = {"School", "Physician", "City", "Practice", "Record"};
+  expected_domains = {"School", "Physician", "Record"};
   BOOST_TEST(nr1.domains == expected_domains, tt::per_element());
 
   BOOST_TEST(tschema.contains("School"));
   T_noisy_relation nr2 = std::get<T_noisy_relation>(tschema["School"]);
   BOOST_TEST(nr2.is_observed);
   BOOST_TEST((nr2.emission_spec.emission == EmissionEnum::bigram_string));
-  // "School" moved to the front of the list.
-  expected_domains = {"School", "City", "Practice", "Physician", "Record"};
+  expected_domains = {"School", "Record"};
   BOOST_TEST(nr2.domains == expected_domains, tt::per_element());
 
   BOOST_TEST(tschema.contains("Degree"));
   T_noisy_relation nr3 = std::get<T_noisy_relation>(tschema["Degree"]);
   BOOST_TEST(nr3.is_observed);
   BOOST_TEST((nr3.emission_spec.emission == EmissionEnum::bigram_string));
-  // "School", "Physician" moved to the front of the list.
-  expected_domains = {"School", "Physician", "City", "Practice", "Record"};
+  expected_domains = {"School", "Physician", "Record"};
   BOOST_TEST(nr3.domains == expected_domains, tt::per_element());
 
   BOOST_TEST(tschema.contains("City"));
   T_noisy_relation nr4 = std::get<T_noisy_relation>(tschema["City"]);
   BOOST_TEST(nr4.is_observed);
   BOOST_TEST((nr4.emission_spec.emission == EmissionEnum::bigram_string));
-  // "City" moved to the front of the list.
-  expected_domains = {"City", "Practice", "School", "Physician", "Record"};
+  expected_domains = {"City", "Record"};
```

Comment:
Similar to above, I think "Practice" should be here too (same for the domains of "State").